atef/check.py

```diff
@@ -631,6 +648,285 @@ def get_by_tag(self, *tags: str) -> Generator[Configuration, None, None]:
             if tag_set.intersection(set(config.tags or [])):
                 yield config
 
+    @classmethod
+    def from_json(cls, filename: AnyPath) -> ConfigurationFile:
```

Comment:
Good call on these, I'll switch the gui to use these methods instead of re-implementing them